crates/flowctl/src/controlplane.rs

```diff
@@ -50,27 +66,98 @@ pub(crate) fn new_client(config: &Config) -> anyhow::Result<Client> {
     }
 }
 
-pub fn configure_new_access_token(ctx: &mut CliContext, token: String) -> anyhow::Result<()> {
+pub async fn configure_new_access_token(ctx: &mut CliContext, token: String) -> anyhow::Result<()> {
     // try to catch issues caused by missing or extra data that may have been accidentally copied
-    let email = check_access_token(&token)?;
-    ctx.config_mut().set_access_token(token);
-    println!("Configured access token for user: '{email}'");
+    let jwt = check_access_token(&token)?;
+    ctx.config_mut().set_access_token(token, jwt.sub);
+    let client = ctx.controlplane_client().await?;
+    let refresh_token = api_exec::<RefreshToken>(
+        client.rpc("create_refresh_token", r#"{"multi_use": true, "valid_for": "14d", "detail": "Created by flowctl"}"#)
```

Comment:
Given that this refresh token can eventually be shown in the UI and revoked at will, I think it's reasonable to give it a very long expiration (or even no expiration) by default.